src/components/ActionMenu/ActionMenu.scss

```diff
@@ -1,9 +1,13 @@
 @import '../../styles/common';
 
 .ActionMenu {
-  @include print-hidden;
   // Needed for JS calculations
   width: 100%;
   display: flex;
   justify-content: flex-end;
+
+  @media print {
+    // stylelint-disable-next-line declaration-no-important
+    display: none !important;
```

Comment:
Now that I see this code in context... I wonder if it even needs the important.

Reply:
Totally, I imagine a lot of these are unnecessary. My guess is the original `hidden-when-printing` mixin included `!important` to cover all edge cases. Happy to spin up a follow up issue to assess all `!important`s throughout `polaris-react`.